pkgs/zmap/zlint/registry.yaml

```diff
@@ -4,6 +4,9 @@ packages:
     repo_owner: zmap
     repo_name: zlint
     description: X.509 Certificate Linter focused on Web PKI standards and requirements
+    files:
+      - name: zlint
+        src: "{{.AssetWithoutExt}}/{{.FileName}}"
```

Comment:
You're misunderstanding.
We need to add `files` at `.packages[].files` if the command name is different from the default command name (a last element of package name) because `aqua g` gets command names from `.packages[].files` and don't see `version_overrides`.
`aqua g` adds command names if they are different from the default command name to improve the searchability.

Reply:
Ah... sorry, I definitely misunderstood.
Thank you for pointing that out and correcting it!